db/dcrpg/pgblockchain.go

```
@@ -4139,6 +4164,6 @@ func (pgb *ChainDBRPC) GetChainWork(hash *chainhash.Hash) (string, error) {
 func (pgb *ChainDB) GenesisStamp() int64 {
 	tDef := dbtypes.NewTimeDefFromUNIX(0)
 	// Ignoring error and returning zero time.
-	pgb.db.QueryRowContext(pgb.ctx, internal.SelectGenesisTime).Scan(&tDef)
+	_ = pgb.db.QueryRowContext(pgb.ctx, internal.SelectGenesisTime).Scan(&tDef)
```

silencing error not checked